src/ShareModule/Components/Sidebar/Sidebar.tsx

```diff
@@ -1,7 +1,202 @@
+import AdsIcon from "@mui/icons-material/LocalOffer";
+import BookingsIcon from "@mui/icons-material/Event";
+import FacilitiesIcon from "@mui/icons-material/LocationCity";
+import ChangePasswordIcon from "@mui/icons-material/Lock";
+import LogoutIcon from "@mui/icons-material/ExitToApp";
+import ChangePassword from "../../../AuthModule/Components/ChangePassword/ChangePassword";
+import { Link } from "react-router-dom";
+
+const drawerWidth = 240;
+
+const openedMixin = (theme: Theme): CSSObject => ({
+  width: drawerWidth,
+  transition: theme.transitions.create("width", {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.enteringScreen
+  }),
+  overflowX: "hidden"
+});
+
+const closedMixin = (theme: Theme): CSSObject => ({
+  transition: theme.transitions.create("width", {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.leavingScreen
+  }),
+  overflowX: "hidden",
+  width: `calc(${theme.spacing(7)} + 1px)`,
+  [theme.breakpoints.up("sm")]: {
+    width: `calc(${theme.spacing(8)} + 1px)`
+  }
+});
+
+const DrawerHeader = styled("div")(({ theme }) => ({
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "flex-end",
+  padding: theme.spacing(0, 1),
+  // necessary for content to be below app bar
+  ...theme.mixins.toolbar
+}));
+
+const Drawer = styled(MuiDrawer, {
+  shouldForwardProp: prop => prop !== "open"
+})(({ theme, open }) => ({
+  width: drawerWidth,
+  flexShrink: 0,
+  whiteSpace: "nowrap",
+  boxSizing: "border-box",
+  ...(open && {
+    ...openedMixin(theme),
+    "& .MuiDrawer-paper": openedMixin(theme)
+  }),
+  ...(!open && {
+    ...closedMixin(theme),
+    "& .MuiDrawer-paper": closedMixin(theme)
+  })
+}));
+
+export default function MiniDrawer() {
+  const [open, setOpen] = React.useState(false);
+  const [openDialog, setOpenDialog] = React.useState(false);
+//  open close SideBar ///////////////
+  const handleDrawerOpen = () => {
+    setOpen(true);
+  };
+  const handleDrawerClose = () => {
+    setOpen(false);
+  };
+////////////////////////////////////
+const handleClickOpenDialog = () => {
+  setOpenDialog(true);
+};
+
+const handleCloseDialog = () => {
+  setOpenDialog(false);
+};
+
+
 
-export default function Sidebar() {
   return (
-    <div className='bg-info'>Sidebar</div>
-  )
+    // Modal Dialog for change Password 
+  
+    <>
+      <Dialog
+    open={openDialog}
+    onClose={handleCloseDialog}
+    aria-labelledby="alert-dialog-title"
+    aria-describedby="alert-dialog-description"
+  >
+    <DialogTitle id="alert-dialog-title">
+      {"change Password "}
+    </DialogTitle>
+    <DialogContent>
+      <ChangePassword handleCloseDialog ={handleCloseDialog}/>
+    
+    </DialogContent>
+
+  </Dialog>
+
+    <Box sx={{ display: "flex" }}>
+      <CssBaseline />
+      <Drawer variant="permanent" open={open} sx={{ "& .MuiDrawer-paper": { backgroundColor: "blue" } }} >
```

Comment:
We shouldn't be using fixed colors, we should choose a color from the MUI colors as this will make it easier to change colors later or implement dark/light mode with just a few lines of code.